client/ayon_deadline/plugins/publish/global/collect_usd_pinning_env_vars.py

```diff
@@ -0,0 +1,51 @@
+import pyblish.api
```

Comment:
Shouldn't this be in ayon-usd?

Reply:
OMG it was already here... 😞

Reply:
Yes - eventually. It already exists in `ayon-deadline`. The TODO also describes that this should move out at some points. I'm just moving code around to de-duplicate the logic. So that as we move to clean it up further we just have the one entry point to change.